xnat_tools/dicom_export.py

```diff
@@ -191,22 +121,22 @@ def main(args):
     if not os.path.exists(logs_dir):
         os.makedirs(logs_dir)
 
-    setup_logging(args.loglevel, f"{logs_dir}/export-{log_id}.log")
+    setup_logging(_logger, f"{logs_dir}/export-{log_id}.log", verbose_level=verbose)
 
     export_session_dir = prepare_export_output_path(
         bids_root_dir,
         pi_prefix,
         study_prefix,
         subject_prefix,
         session_prefix,
-        overwrite,
+        overwrite=overwrite,
     )
 
+    # Export
     scans = get_scan_ids(connection, host, session)
-    scans = filter_scans(scans, seqlist=seqlist, skiplist=skiplist)
+    scans = filter_scans(scans, seqlist=includeseq, skiplist=skipseq)
```

Comment:
Why the name change in `includeseq`? I personally prefer the consistency especially if we're passing the list around

Reply:
I think I did it so that the `i` CLI option was more consistent with the name, and then I was lazy to change the actual variable all around. Consistency is good!